Client/Frontend/TabContentsScripts/LoginsHelper.swift

```diff
@@ -29,8 +29,8 @@ class LoginsHelper: TabContentScript {
         self.theme = theme
     }
 
-    func scriptMessageHandlerName() -> String? {
-        return "loginsManagerMessageHandler"
+    func scriptMessageHandlerNames() -> [String]? {
+        return ["loginsManagerMessageHandler"]
```

Comment:
can we call this logInManagerMessageHandler

Reply:
We will need to change code elsewhere. We can do that in a followup if you'd like.